src/Metzli/Renderer/FpdfRenderer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * Copyright 2018 Metzli authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Metzli\Renderer;
+
+use Metzli\Encoder\AztecCode;
+
+class FpdfRenderer implements RendererInterface
+{
+    private $fgColor;
+    private $bgColor;
+    private $pdf;
+
+    private $x;
+    private $y;
+
+    private $s;
+
+    public function __construct(\FPDF $fpdfLikeObject = null, $s = false, $x = false, $y = false, $fgColor = array(0, 0, 0), $bgColor = array(256, 256, 256))

```

Comment:
I like the default being an image with a transparent background. Can we use `null` instead of `[256, 256, 256]` to specify transparency?